### ISSUE TITLE
Channel Buttons Sometimes Go to IPTV Channels

### ISSUE THREAD

**Issue by thehauntedmattress** [merdok/homebridge-webos-tv#83]
First off thank you for the channel buttons.  However, I noticed that some single digit channel buttons tune the TV to IPTV based channels instead of over the air antenna based ones.  For example I have a button that is channel 9 (over the air channel 9-1).  However, when I press that button, it sometimes (not every time) takes me to IPTV channel 999.  I was wondering if there could be a fix for this.  Thanks.

**Comment by merdok** [comment]
I am not sure how this is possible. Especially the randomness is what confuses me. Are you sure that this is not something how your TV configured?
I have only cable channels on my TV so i cannot really test it...

**Comment by merdok** [comment]
Could you try the latest update?

**Comment by thehauntedmattress** [comment]
Updated.  Issue is still there.

**Comment by merdok** [comment]
Ok, then this is most probably a setting issue with your TV. You would need to check your TV settings and further investigate this issue. Please let me know when you find which tv setting it was.

**Comment by thehauntedmattress** [comment]
I have figured it out.  It was completely on me.  Apparently even though I could manually on the remote tune the TV to 9-1, it wasn’t on my channel list if I used the channel up an down buttons.  Therefore the TV didn’t think 9-1 existed, so it would go to the next 9 channel.  So I guess maybe in your description put “Only channels that are listed on your TV’s channel list can be tuned to.  Make sure to do a thorough antenna channel scan before using channel buttons.”
